packages/client/src/components/links/favLinksContainer.tsx

```diff
@@ -19,6 +20,7 @@ export default function FavLinksContainer({ links, title }: Props) {
         container.scrollLeft += event.deltaY * 0.2;
       }
     };
+    getLinks().then((data) => setLinks(data));
```

Comment:
would be better if the backend could be called from a server component. Let's talk tonight :D